godtools/Views/TractElements/TractPageContainer.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  TractPageContainer.swift
+//  godtools
+//
+//  Created by Pablo Marti on 11/7/17.
+//  Copyright © 2017 Cru. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import SWXMLHash
+
+class TractPageContainer: BaseTractElement {
+    
+    override var properties: TractProperties {
+        get {
+            return (self.parent?.properties)!
+        }
+        set { }
+    }
+    
+    // MARK: - Setup
+    static var marginBottom: CGFloat {
+        return UIDevice.current.iPhoneX() ? CGFloat(44.0) : CGFloat(0.0)
```

Comment:
These should be made constants to. They are the same values as the status bar height, but they are use for different things, right? I think it's correct to make another set of constants in this class.

At some point we should try to consolidate our constants somehow, but that could be a separate task for another time.